pkgs/applications/audio/musescore/default.nix

```diff
@@ -54,6 +54,9 @@ in stdenv'.mkDerivation rec {
     rev = "v${version}";
     sha256 = "sha256-CqW1f0VsF2lW79L3FY2ev+6FoHLbYOJ9LWHeBlWegeU=";
   };
+  patches = [
+    ./darwin-fix-install.patch
```

Comment:
Can you please add a comment on why this patch is required?

Reply:
> Can you please add a comment on why this patch is required?

I will, once I see that it works.